site/docs/status.md

```diff
@@ -0,0 +1,362 @@
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Implementations Status
+
+Apache iceberg now has implementations of the iceberg spec in multiple languages. This page provides a summary of the
+current status of these implementations.
+
+## Versions
+
+This section describes the versions of each implementation that are being tracked in this page.
+
+| Language  | Version |
+|-----------|---------|
+| Java      | 1.7.1   |
+| PyIceberg | 0.7.0   |
+| Rust      | 0.3.0   |
+| Go        | 0.1.0   |
+
+## Data Types
+
+| Data Type      | Java | PyIceberg | Rust | Go |
+|----------------|------|-----------|------|----|
+| boolean        | Y    | Y         | Y    | Y  |
+| int            | Y    | Y         | Y    | Y  |
+| long           | Y    | Y         | Y    | Y  |
+| float          | Y    | Y         | Y    | Y  |
+| double         | Y    | Y         | Y    | Y  |
+| decimal        | Y    | Y         | Y    | Y  |
+| date           | Y    | Y         | Y    | Y  |
+| time           | Y    | Y         | Y    | Y  |
+| timestamp      | Y    | Y         | Y    | Y  |
+| timestamptz    | Y    | Y         | Y    | Y  |
+| timestamp_ns   | Y    | Y         | Y    | Y  |
+| timestamptz_ns | Y    | Y         | Y    | Y  |
+| string         | Y    | Y         | Y    | Y  |
+| uuid           | Y    | Y         | Y    | Y  |
+| fixed          | Y    | Y         | Y    | Y  |
+| binary         | Y    | Y         | Y    | Y  |
+
+## Data File Formats
+
+| Format  | Java | PyIceberg | Rust | Go |
+|---------|------|-----------|------|----|
+| Parquet | Y    | Y         | Y    | Y  |
+| ORC     | Y    | N         | N    | N  |
+| Puffin  | Y    | N         | N    | N  |
+
+## File IO
+
+| Storage              | Java | PyIceberg | Rust | Go |
+|----------------------|------|-----------|------|----|
+| Local Filesystem     | Y    | Y         | Y    | Y  |
+| Hadoop Filesystem    | Y    | Y         | Y    | Y  |
+| Aws S3               | Y    | Y         | Y    | Y  |
+| Google Cloud Storage | Y    | Y         | Y    | Y  |
+| Memory Fs            | Y    | Y         | Y    | Y  |
+
+## Table Update Operations
+
+### Table Spec V1
+
+| Operation                   | Java | PyIceberg | Rust | Go |
+|-----------------------------|------|-----------|------|----|
+| Update schema               | Y    | N         | N    | N  |
+| Update partition spec       | Y    | N         | N    | N  |
+| Update table properties     | Y    | Y         | Y    | N  |
+| Replace sort order          | Y    | N         | N    | N  |
+| Update table location       | Y    | N         | N    | N  |
+| Append data files           | Y    | N         | N    | N  |
+| Rewrite files               | Y    | N         | N    | N  |
+| Rewrite manifests           | Y    | N         | N    | N  |
+| Overwrite files             | Y    | N         | N    | N  |
+| Row delta                   | Y    | N         | N    | N  |
```

Comment:
Row delta is v2 spec. So, I don't understand mentioning it under v1 spec.

Reply:
I think it is a copy paste :)